life/modernized/27-chapter-27.adoc

```diff
@@ -574,7 +583,7 @@ This, I say,
 is as the root of the goodly tree of doctrines that grew and branched out from it,
 which I shall now mention in their natural and experimental order.
 
-First, Repentance from dead works to serve the living God.
```

Comment:
So this paragraph begins with "first", but there is no "secondly" that i could see, but then a paragraph or two below there is a "thirdly". So I just left as is.